### DISCUSSION TITLE
Add an option to create an output folder for encoded files and *not* a ZIP file.

### DISCUSSION THREAD

**Discussion by Nemgathos** [MeViMo/youbit#12] [Ideas]
It would be good if there were an option in `YouBit`’s `encode` method that allows the user to choose between returning the standard output (a `ZIP` file) and writing the contents directly to a specific folder. Right now, it is only possible to create a `ZIP` file, unzip it afterwards and then delete the `ZIP` file. This is pretty cumbersome and not very good when it comes to terms like speed and sparing your HDD or SSD from a lot of reading and writing operations.

**Comment by flonle** [comment]
This could defintely be added. I don't have time for this right now but I'd be happy to approve a PR.

**Reply by Nemgathos** [reply]
What do you mean by “PR”? A programmer? I haven’t used `Python` in the past and have only watched one or two tutorial videos so far. But I know that it is not difficult to learn `Python`. I have `Java`, `C`, `R` and various `BASIC` dialect skills. Also, time is an issue for me as well. First, I should study the code and understand what it is doing at the relevant lines that should be changed in order to add such a feature.

**Reply by flonle** [reply]
PR means Pull Request.
You're under no obligation to make contributions. I'm just saying I don't want to write it, but if you do, I'd approve the code.

**Reply by Nemgathos** [reply]
Ah, the good old pull request, I see. As I wrote, I do not know whether I have time for that either. But if I understand how the program works and understand what the `Python` code is doing, I might consider editing your code in a separate pull request.
